### ISSUE TITLE
unknown property 'SDK_REGISTRY_TOKEN' error

### ISSUE THREAD

**Issue by pengju114** [mapbox/mapbox-maps-android#2453]
When building the project, the build system generates the following error:
Could not get unknown property 'SDK_REGISTRY_TOKEN' for root project 'Mapbox Maps Android' of type org.gradle.api.Project.

And I have placed a property “SDK_REGISTRY_TOKEN” and set it's value to my token in the local.properties file, But the error still there. Some one can help? thanks.

**Comment by pengju114** [comment]
> Thank you for the question. You can follow this guide to set up `SDK_REGISTRY_TOKEN` https://docs.mapbox.com/android/maps/guides/install/ Could you try to set it up in the Environment variable?

Thank you for your suggestion, but I tried it and it doesn’t seem to work.

**Comment by flasher297** [comment]
Could you please give more context? Do you try to compile example app from this repo?

**Comment by pengju114** [comment]
@flasher297 yes, i want to run the example downloaded from this repo on the android device, for some technology research. and now my project of the example still building fail. i have config the environment  following your guidance, but it doesn't work.

**Comment by flasher297** [comment]
@pengju114 thank you for the additional explanations.
We'll try to verify the ability to compile our public examples in this repo and come back to you with a report.

**Comment by pengju114** [comment]
thank you very much. and below is the error screen shot:
![screenshot-20240926-121757](https://github.com/user-attachments/assets/3cdc167c-81a5-41b6-8232-a6627d0aa43a)

**Comment by pariahware** [comment]
The tutorial says to use $MAPBOX_ACCESS_TOKEN as the environment variable, but the project uses $SDK_REGISTRY_TOKEN. I just made both with the same key.
